edgedbpkg/pgext/pgvector/__init__.py

```diff
@@ -23,3 +25,8 @@ class PgVector(pgext.PostgresCExtension):
     @property
     def supports_out_of_tree_builds(self) -> bool:
         return False
+
+    def get_build_env(self, build: targets.Build, wd: str) -> packages.Args:
```

Comment:
You probably want `get_make_args` here instead as I'm not sure if `make` will pick a variable from the environment in this case.

Reply:
Roger